### PR TITLE
chore(sel): move Summary() call into Service and Job methods

### PR DESCRIPTION
**Pull request by huanjani**
Fixes errors introduced in https://github.com/aws/copilot-cli/pull/1475.
`Summary()` is no longer being called in `NewWorkspaceSelect()` because the necessary workspace will have been created by the time it's needed, so `copilot init` e2e tests should no longer fail.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by iamhopaul123 — COMMENTED** [review]
Awesome! Looks good to me! Thank you for making this efforts. Could you double check we are not using `workspace only` workloads (doesn't matter if it is in config store or not). And also could you run e2e tests against this PR?

**Comment by huanjani** [comment]
Yep-- ran just the `init` suite and
```
Ran 9 of 9 Specs in 1033.984 seconds
SUCCESS! -- 9 Passed | 0 Failed | 0 Pending | 0 Skipped
PASS

Ginkgo ran 1 suite in 30m29.5500771s
Test Suite Passed
```

**Comment by iamhopaul123** [comment]
> Yep-- ran just the `init` suite and
> 
> ```
> Ran 9 of 9 Specs in 1033.984 seconds
> SUCCESS! -- 9 Passed | 0 Failed | 0 Pending | 0 Skipped
> PASS
> 
> Ginkgo ran 1 suite in 30m29.5500771s
> Test Suite Passed
> ```

Looks good! I'll approve it once we confirm all suites pass.

**Review by iamhopaul123 — APPROVED** [review]
:shipit:

File: internal/pkg/term/selector/selector.go [hunk 1]
```diff
@@ -155,14 +155,9 @@ func NewConfigSelect(prompt Prompter, store ConfigLister) *ConfigSelect {
 // NewWorkspaceSelect returns a new selector that chooses applications and environments from the config store, but
 // services from the local workspace.
 func NewWorkspaceSelect(prompt Prompter, store ConfigLister, ws WorkspaceRetriever) (*WorkspaceSelect, error) {
```
Comment (efekarakus) [inline internal/pkg/term/selector/selector.go]: Can we remove the error from the outputs since now this cannot error anymore :)

**Review by efekarakus — APPROVED** [review]

**Pull request merged**